### DISCUSSION TITLE
Being able to search for a beatmap that someone is currently playing/editing in the in-game player search box

### DISCUSSION THREAD

**Discussion by Senuashi** [ppy/osu#24035] [Ideas] [answered]
A player would be able to search for any beatmap in the player search box (F9) and as a result, a list of players who are either playing or editing the beatmap would be shown (instead of being able to search only for players).

The main reason I would like this function added is to have an easier time finding another players for beatmap collaborations/guest difficulties. Being able to simply type the name of a beatmap and see who is currently editing one could prove to be very helpful.

Although the previous reason is far more significant, another reason to add this function could be if a player wanted to see someone play a certain beatmap (for example if a mapper wanted to see if their map is currently being played by another player).

Would look something like this:
![xfcgxf](https://github.com/ppy/osu/assets/82316882/57654982-21ac-42b8-9e6e-ed4aec61888b)

Of course, this is just an idea, all of this could be just too niche for it to be implemented into the game.

**Comment by peppy** [comment]
We're not event showing the current beatmap being played yet, so that would need to come first. But it should be easy enough to add this when ready.

**Reply by peppy** [reply, marked as answer]
Tracking at https://github.com/ppy/osu/issues/24042
